service/controller/chart/resource/release/resource.go

```diff
@@ -322,3 +322,19 @@ func toReleaseState(v interface{}) (ReleaseState, error) {
 
 	return *releaseState, nil
 }
+
+func convertFloat(m map[string]interface{}) {
+	for k, val := range m {
+		switch val.(type) {
+		case float64:
+			converted := int(val.(float64))
+			if val.(float64) == float64(converted) {
```

Comment:
If the decimal is the same without floating points, that means it should be an integer type.